### ISSUE TITLE
Chart does not render on polyfilled browsers

### ISSUE THREAD

**Issue by tomalec** [StarcounterApps/KitchenSink#76]
### Steps to reproduce

1. Run KitchenSink
2. Open [/KitchenSink/Chart](http://localhost:8080/KitchenSink/Chart)

### Expected result
Chart should be shown

### Actual result
...it's not

### Technical information

Error is thrown to console:
```log
15:07:00.297 TypeError: this.scale is undefined
.draw()Chart.js:2781
helpers.animationLoop/animationFrame()Chart.js:697
1Chart.js:2781:4
```

##### Involved apps' version

 - StarcounterSamples/KitchenSink@4f6309355854cf298a1decba14c5ea298dae35b5 (current HEAD of develop)
 
##### Browsers affected

- [ ] Chrome
- [x] Firefox
- [x] Edge
- [x] Safari 9
- [?] Safari 8
- [x] IE 11

### Screenshot
![image](https://cloud.githubusercontent.com/assets/17435/20060575/269e79ce-a4fc-11e6-8ef6-f34c3e4f28a4.png)

**Comment by warpech** [comment]
@PatrykSzwer could you pls take a look at this?

**Comment by PatrykSzwer** [comment]
It's still reproducing. Trying to fix this now without results yet.

**Comment by warpech** [comment]
Maybe upgrading to the current version helps?

**Comment by PatrykSzwer** [comment]
I've made PR (#127) with fix on that. I've rebuild entire way that chart on chartPage works, by creating custom element.

**Comment by PatrykSzwer** [comment]
Fixed in #127. Closing.
